docs/components/connectors/out-of-the-box-connectors/amazon-sns.md

```diff
@@ -40,6 +40,7 @@ To make your Amazon SNS Connector for sending messages executable, take the foll
 4. In the **Input message data** section, fill out the field **Message** with the data you would like to publish to the topic. The field requires FEEL input.
 5. (Optional) In the **Input message data** section, fill out the field **Message attributes** to set optional message metadata. This field requires FEEL input. Refer to the relevant [appendix](#what-are-the-message-attributes-and-how-can-i-set-them) section to find out more about this field.
 6. (Optional) In the **Input message data** section, fill out the field **Subject** to set optional message subject. FEEL input of the field is optional. Length must be less than 100 characters.
+7. (FIFO only) For a FIFO type topic in AWS SNS, a **Message Group ID** is required. This ID ensures that messages within the same group are delivered in sequence. The [AWS SNS documentation on FIFO topics](https://docs.aws.amazon.com/sns/latest/dg/sns-fifo-topics.html) provides more details on Message Group ID usage. Additionally, an optional **Message Deduplication ID** can be provided. This is useful for message deduplication in FIFO topics and its necessity depends on the [deduplication settings of your SNS FIFO topic](https://docs.aws.amazon.com/sns/latest/dg/sns-message-deduplication.html). The Message Deduplication ID helps ensure AWS SNS does not resend the same message within the deduplication interval.
```

Comment:
We just need to spell out as "Amazon SNS" rather than "AWS SNS" for brand naming purposes here.